### DISCUSSION TITLE
How can I display blog posts excerpt?

### DISCUSSION THREAD

**Discussion by xsinger** [datenstrom/community#280] [Ask a question] [answered]
Yellow can display posts excerpt?

For example, 200 words per article.

thx!

**Comment by markseuffert** [comment, marked as answer]
Yes, you can use the following in your snippets: 

`<?php echo $yellow->toolbox->createTextDescription($page->getContent(), 150) ?>`. 
The first example will give you the first 150 characters of the page content, HTML is removed.

`<?php echo $yellow->toolbox->createTextDescription($page->getContent(), 500, false) ?>`. 
The second example will give you the first 500 characters of the page content,  HTML is not removed.

Hope this helps. 😄

**Comment by BsNoSi** [comment]
If you want to display a _logical_ excerpt, you can use `[--more--]` as separator. It cuts off the text before it from the rest in article listings. The command itself is invisible and replaced by `Read more…` (or the respective translation).

**Comment by xsinger** [comment]
It works perfectly, thank you!
